nvflare/private/fed/client/client_run_manager.py

```diff
@@ -263,7 +249,7 @@ def _to_aux_msg_targets(self, target_names: List[str]):
         if isinstance(target_names, str):
             if target_names == SiteType.ALL:
                 msg_targets = [AuxMsgTarget.server_target()]
-                for _, c in self.all_clients.items():
+                for c in self.all_clients:
```

Comment:
Not sure about this loop. If all_clients is a dict, this is looping through the keys.

Reply:
nm. Saw this is changed to a list now.